src/sandstorm/sandstorm-http-bridge.c++

```diff
@@ -1300,17 +1272,14 @@ private:
   kj::AutoCloseFd identitiesDir;
   kj::AutoCloseFd trashDir;
 
-  // These maps store information relating to active sessions. `sessions` always has an entry
-  // for any active session that is handled by the app. `requests` and `offers` will only
-  // have an entry for request and offer sessions, respectively.
-  //
-  // In all cases the keys are owned by the session object, which is responsible for deleting
-  // the session from these maps (via eraseSession) on cleanup. For `sessions`, the value
-  // is also owned by the session. For `requests` and `offers`, the value is owned by the map
-  // itself.
-  std::map<kj::StringPtr, SessionContext::Client&> sessions;
-  std::map<kj::StringPtr, kj::Own<SandstormHttpBridge::GetSessionRequestResults::Reader>> requests;
-  std::map<kj::StringPtr, kj::Own<SandstormHttpBridge::GetSessionOfferResults::Reader>> offers;
+  struct SessionRecord {
+    SessionRecord(const SessionRecord& other) = delete;
+    SessionRecord(SessionRecord&& other) = default;
+
+    SessionContext::Client& sessionCtx;
+    kj::Own<SessionInfo::Reader>& sessionInfo;
```

Comment:
Nitpick, but this is now a reference to a kj::Own, which seems a little redundant. Why not just declare this as:

    SessionInfo::Reader sessionInfo;

?

Reply:
Done.